catalyst/core/callbacks/metrics.py

```diff
@@ -21,7 +21,7 @@ def __init__(
         multiplier: float = 1.0,
         **metrics_kwargs,
     ):
-        super().__init__(CallbackOrder.Metric)
+        super().__init__(order=CallbackOrder.Metric, node=CallbackNode.All)
```

Comment:
master node?

Reply:
because you need to measure metric on all nodes